### PR TITLE
Personal/miketur/remove dfci pkg

### PR DESCRIPTION
**Pull request by mikeytdisco**
# Preface

Please ensure you have read the [contribution docs](https://github.com/microsoft/mu/blob/master/CONTRIBUTING.md) prior
to submitting the pull request. In particular,
[pull request guidelines](https://github.com/microsoft/mu/blob/master/CONTRIBUTING.md#pull-request-best-practices).

## Description

DfciPkg will be moving to a new repo, mu_feature_dfci.  This PR is part of that process.

For each item, place an "x" in between `[` and `]` if true. Example: `[x]`.
_(you can also check items in the GitHub UI)_

- [ ] Impacts functionality?
- [ ] Impacts security?
- [x] Breaking change?
- [ ] Includes tests?
- [ ] Includes documentation?

## How This Was Tested

Only tested manually using the QuemuQ35 platform.

## Integration Instructions

A (TBD) PR for mu_tiano_patforms will publish the integration instructions.

### CODE REVIEW

File: MsCorePkg/MsCorePkg.ci.yaml [hunk 1]
```diff
@@ -104,7 +104,9 @@
             "POLICYREAD",
             "POLICYWRITE",
             "SQRTUNSIGNED",
-            "VARPOL"
+            "VARPOL",
+            "SNP's",
```
Comment (kuqin12) [inline MsCorePkg/MsCorePkg.ci.yaml:108]: Why do we have new entries after removing code?

File: MsCorePkg/MsCorePkg.ci.yaml [hunk 2]
```diff
@@ -104,7 +104,9 @@
             "POLICYREAD",
             "POLICYWRITE",
             "SQRTUNSIGNED",
-            "VARPOL"
+            "VARPOL",
+            "SNP's",
+            UEFI's
```
Comment (kuqin12) [inline MsCorePkg/MsCorePkg.ci.yaml]:
```suggestion
            "UEFI's"
```

**Review by kuqin12 — APPROVED** [review]

**Comment by kuqin12** [comment]
Also might want to update the PR title.

File: MsCorePkg/MsCorePkg.ci.yaml (hunk 1 above)
Comment (mikeytdisco) [reply MsCorePkg/MsCorePkg.ci.yaml:108]: MsCorePkg\MacAddressEmulationDxe\MacAddressEmulationDxe.c has the offending word.  Change in spellcheck, I assume.

**Review by makubacki — APPROVED** [review]

**Comment by makubacki** [comment]
> Also might want to update the PR title.

@kuqin12 & @mikeytdisco 

I didn't realize auto merge was set and Kun's comment was not a review comment which would hold the PR. So, when I approved, it automatically merged.

Given this is an important PR, I'm open to following up with something like an empty commit that says what was previously removed. Please let me know if you'd prefer this and I can manually push it.

**Comment by kuqin12** [comment]
> > Also might want to update the PR title.
> 
> @kuqin12 & @mikeytdisco
> 
> I didn't realize auto merge was set and Kun's comment was not a review comment which would hold the PR. So, when I approved, it automatically merged.
> 
> Given this is an important PR, I'm open to following up with something like an empty commit that says what was previously removed. Please let me know if you'd prefer this and I can manually push it.

I think we can do that.

To prevent more misses like this, is there any way we can pose "conversation"-like comments to PR here? So that the PR will put on held?

**Comment by makubacki** [comment]
> > > Also might want to update the PR title.
> > 
> > 
> > @kuqin12 & @mikeytdisco
> > I didn't realize auto merge was set and Kun's comment was not a review comment which would hold the PR. So, when I approved, it automatically merged.
> > Given this is an important PR, I'm open to following up with something like an empty commit that says what was previously removed. Please let me know if you'd prefer this and I can manually push it.
> 
> I think we can do that.
> 
> To prevent more misses like this, is there any way we can pose "conversation"-like comments to PR here? So that the PR will put on held?

Typically, the comment needs to be submitted as a "review comment" to hold the PR. Those will have the ability to be marked resolved. I think this is one of a number of UI design issues with code reviews in GitHub.

Here's the PR to clarify this PR in history: https://github.com/microsoft/mu_plus/pull/157

**Comment by kuqin12** [comment]
> > > > Also might want to update the PR title.
> > > 
> > > 
> > > @kuqin12 & @mikeytdisco
> > > I didn't realize auto merge was set and Kun's comment was not a review comment which would hold the PR. So, when I approved, it automatically merged.
> > > Given this is an important PR, I'm open to following up with something like an empty commit that says what was previously removed. Please let me know if you'd prefer this and I can manually push it.
> > 
> > 
> > I think we can do that.
> > To prevent more misses like this, is there any way we can pose "conversation"-like comments to PR here? So that the PR will put on held?
> 
> Typically, the comment needs to be submitted as a "review comment" to hold the PR. Those will have the ability to be marked resolved. I think this is one of a number of UI design issues with code reviews in GitHub.
> 
> Here's the PR to clarify this PR in history: #157

Hmm, your PR is actually a perfect example, how do we leave a review comment without any file change in it?

**Pull request merged**